src/bp/ui-admin/src/Pages/Confusion/index.jsx

```diff
@@ -158,17 +158,24 @@ class Confusion extends Component {
         <div key={`conf-${confusion.botId}`}>
           <h3>{confusion.botId}</h3>
           {confusion.confusions
-            .filter(x => _.get(x, 'matrix.intents.all.f1'))
+            .filter(x => _.get(x, 'matrix.all.f1'))
             .filter(data =>
               _.isEmpty(_.difference(this.getValueForAttributes(allFilterFields)(data), this.state.select))
             )
             .map((data, i) => (
               <div key={data.hash + i}>
                 <h6>
-                  [{data.lang.toUpperCase()}] - F1: <strong>{data.matrix.intents.all.f1.toFixed(2)}</strong>
+                  [{data.lang.toUpperCase()}] -{' '}
+                  {['f1', 'recall', 'precision'].map(field => (
+                    <strong>{field.toString() + ': ' + data.matrix.all[field].toFixed(2) + ', '}</strong>
```

Comment:
Could you post a screenshot for you PR? I suspect there might be a extra comma after after precision :

precision: 555,

Reply:
There is indeed an extra ',' indeed